test/functional/p2p_feefilter.py

```diff
@@ -83,27 +77,27 @@ def test_feefilter_forcerelay(self):
     def test_feefilter(self):
         node1 = self.nodes[1]
         node0 = self.nodes[0]
+        miniwallet = MiniWallet(node1)
+        miniwallet.generate(5)
```

Comment:
I think this can just be `generate(1)` since the output UTXO can always be used in subsequent tests.

Could probably do with a comment so it's obvious why two different generate commands are being used here.

Reply:
Not sure if we want package fee rats in this test. I've kept the previous behavior (packets of size 1)

Reply:
I still think a comment would be useful for people reading this file for the first time. Not essential though.

Reply:
added comment